style.css

```diff
@@ -1295,7 +1347,7 @@ h1.big-heading {
   /* position:absolute; */
   right: 30%;
   margin-bottom: 50px;
-  margin-top: -50px;
+ 
```

Comment:
why is this removed?

Reply:
It was done to make it responsive, I'll add that again.